cli/cmd/profile.go

```diff
@@ -34,15 +35,38 @@ type profileOptions struct {
 	openAPI   string
 }
 
-func newProfileOptions() *profileOptions {
-	return &profileOptions{
+func newProfileOptions() profileOptions {
+	return profileOptions{
 		name:      "",
 		namespace: "default",
 		template:  false,
 		openAPI:   "",
 	}
 }
 
+func validateOptions(options profileOptions) error {
```

Comment:
TIOLI -- for some of our other commands, the convention has been to add a `validate()` func on the options struct itself, rather than passing it as a param. Something like:

```diff
diff --git a/cli/cmd/profile.go b/cli/cmd/profile.go
index c68bf5e9..4bfcfa7e 100644
--- a/cli/cmd/profile.go
+++ b/cli/cmd/profile.go
@@ -44,7 +44,7 @@ func newProfileOptions() profileOptions {
 	}
 }
 
-func validateOptions(options profileOptions) error {
+func (options *profileOptions) validate() error {
 	outputs := 0
 	if options.template {
 		outputs++
@@ -96,7 +96,7 @@ Example:
 		RunE: func(cmd *cobra.Command, args []string) error {
 			options.name = args[0]
 
-			err := validateOptions(options)
+			err := options.validate()
 			if err != nil {
 				return err
 			}
```